test/controllers/github_webhooks_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class GithubWebhooksControllerTest < ActionDispatch::IntegrationTest
```

Comment:
If we're not going to write tests here we should remove this file entirely.

Reply:
Okay @undergroundwebdesigns , Removed this file